lib/kafo/kafo_configure.rb

```diff
@@ -193,13 +193,18 @@ def execute
         @progress_bar = config.app[:colors] ? ProgressBars::Colored.new : ProgressBars::BlackWhite.new
       end
 
-      unless skip_checks_i_know_better?
-        unless SystemChecker.check
-          puts "Your system does not meet configuration criteria"
+      if checks_only? || !skip_checks_i_know_better?
```

Comment:
Technically, `checks_only?` is not required here?
Unless you expect people to run `--checks-only --skip-checks-i-know-better`?

Reply:
I don't expect it, but its **possible** and I wanted the outcome to make sense.

Reply:
heh, fair